src/test/java/BoardTest.java

```diff
@@ -6,11 +6,23 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 class BoardTest {
+
     @Test
     void returnsGrid() {
         var grid = new Grid
-                (new ArrayList<String>(Arrays.asList("1", "2", "3", "4", "5", "6", "7", "8", "9")));
+                (new ArrayList<>(Arrays.asList
+                        ("1", "2", "3", "4", "5", "6", "7", "8", "9")));
         var board = new Board(grid);
         assertEquals(board.getGrid(), grid);
     }
+
+    @Test
+    void addsMarkToGrid() {
+        var grid = new Grid
+                (new ArrayList<>(Arrays.asList
+                        ("1", "2", "3", "4", "5", "6", "7", "8", "9")));
+        var board = new Board(grid);
+        board.addMarkToGrid(1, "x");
```

Comment:
Looking at this test, does `board.addMarkToGrid` seem a bit too specific? I mean if you were describing what it does, what would you say? I would probably go for something like:`board.takeTile(1, 'x')` or board.takeSquare(1, 'x')`?

Reply:
Thanks, I'll change this